src/components/custom-sdk/template/HMRC_ODX_GDSTaskListTemplate/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { Fragment } from 'react';
+import { Grid } from '@pega/cosmos-react-core';
+import { useTranslation } from 'react-i18next';
+import type { PConnProps } from '@pega/react-sdk-components/lib/types/PConnProps';
+import StyledHmrcOdxGdsTaskListTemplateWrapper from './styles';
+import './TaskListTemplate.css';
+
+interface HmrcOdxGdsTaskListTemplateProps extends PConnProps {
+  // If any, enter additional props that only exist on this componentName
+  children: Array<any>;
+  cssClassHook: string;
+  NumCols: string;
+}
+
+export default function HmrcOdxGdsTaskListTemplate(props: HmrcOdxGdsTaskListTemplateProps) {
+  const { children, cssClassHook, NumCols, getPConnect } = props;
+  const nCols = parseInt(NumCols, 8);
+  const { t } = useTranslation();
+  const context = getPConnect().getContextName();
+  const caseInfo = getPConnect().getCaseSummary();
+  const data = caseInfo.content.CaseTaskList;
+
+  let totalSections = 0;
+  let completedSections = 0;
+
+  // Loop through the data to determine the number of sections in total and how many are flagged as complete.
+  data.map(task => {
+    totalSections += 1;
+    if (task.IsTaskComplete) {
+      completedSections += 1;
+    }
+  });
```

Comment:
getting eslint issue that map function should return

Reply:
Thanks @gautamkumar-hmrc - I found this earlier as well.  Will amend.

Reply:
Have updated to forEach as not returning an array